src/Helpers/helpers.php

```diff
@@ -94,3 +94,63 @@ function mollie_object_to_money(object $object)
         return decimal_to_money($object->value, $object->currency);
     }
 }
+
+if (! function_exists('starts_with')) {
+    /**
+     * Determine if a given string starts with a given substring.
+     *
+     * @param  string  $haystack
+     * @param  string|array  $needles
+     * @return bool
+     */
+    function starts_with($haystack, $needles)
+    {
+        foreach ((array) $needles as $needle) {
+            if ($needle !== '' && substr($haystack, 0, strlen($needle)) === (string) $needle) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+}
+
+if (! function_exists('ends_with')) {
+    /**
+     * Determine if a given string ends with a given substring.
+     *
+     * @param  string  $haystack
+     * @param  string|array  $needles
+     * @return bool
+     */
+    function ends_with($haystack, $needles)
+    {
+        foreach ((array) $needles as $needle) {
+            if (substr($haystack, -strlen($needle)) === (string) $needle) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+
+
+    if (! function_exists('snake_case')) {
```

Comment:
Seems like you've put this inside the if-statement above. Shouldn't this be outside the `if (! function_exists('ends_with'))`?

Reply:
Ups!!! Thanks

Reply:
@ciungulete are you able to fix this (and push to this branch)? Otherwise I'll pick it up.